### PR TITLE
fix bug in average function for Top accuracy by class AverageMeter

### PR DESCRIPTION
**Pull request by kaijieshi7**
求Top-1，Top-5准确率的过程中涉及到了平均每个batch准确率的问题。
之前的方法因为last batch和之前的batch不一定相同，会导致换个batch size得到的Top-1和Top-5不相同。
更新之后，使用全部数据集来求平均数。

### CODE REVIEW

File: benchmark.py [hunk 1]
```diff
@@ -62,24 +61,39 @@ def __init__(
...
-        correct_k = correct[:k].contiguous().view(-1).float().sum(0)
-        res.append(correct_k / batch_size * 100.0)
-    return res
+    return [correct[:k].reshape(-1).float().sum(0) * 100. / batch_size for k in topk]
 
 
 def main(args):
-    model = ModelCreator.create_model(args.model, pretrained=True)
+    model = ModelCreator.create_model(args.model, pretrained=False)
+    model.load_state_dict(flow.load('/home/kaijie/Documents/code/of/vision_git/vision/flowvision/models/ghostnet.zip'))
```
Comment (rentainhe) [inline benchmark.py:96]: 这一行可以删一下，在benchmark.sh这个文件里传ghostnet参数就可以了

- https://github.com/Oneflow-Inc/vision/blob/2826110363024eba64284c7ec71a43dfd8ff2889/benchmark.sh#L3

File: benchmark.py [hunk 2]
```diff
@@ -39,10 +39,9 @@ def __init__(
                 ]
             )
         else:
-            size = int((256 / 224) * image_size)
             transform = transforms.Compose(
                 [
-                    transforms.Resize(size, interpolation=3)  # 3: bibubic
+                    transforms.Resize(256, interpolation=3)  # 3: bibubic
```
Comment (rentainhe) [inline benchmark.py:44]: 这里对齐一下最新master里的DEFAULT_CROP_PCT的形式，不用改为固定值，可以通过CROP-PCT这个参数控制缩放比率

参考代码：https://github.com/Oneflow-Inc/vision/blob/2826110363024eba64284c7ec71a43dfd8ff2889/benchmark.py#L46

Comment (rentainhe) [reply benchmark.py:44]: 默认的情况下`crop-size=0.875`，也就是 scale-size等于`224 -> 256`

**Comment by Ldpe2G** [comment]
有冲突要解决

File: benchmark.py (hunk 2 above)
Comment (kaijieshi7) [reply benchmark.py:44]: 最新的分支就是固定值啊

**Review by rentainhe — APPROVED** [review]
